app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -24,6 +24,23 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         addPreferencesFromResource(R.xml.settings);
+
+        setupPreferences(getPreferenceScreen());
+    }
+
+    private void setupPreferences(PreferenceGroup group) {
+        for (int i = 0; i < group.getPreferenceCount(); i++) {
+            final Preference preference = group.getPreference(i);
+
+            if (preference instanceof PreferenceGroup) {
+                setupPreferences((PreferenceGroup) preference);
+            }
+
+            if (preference.getTitle().equals(getString(R.string.preference_about))) {
```

Comment:
Another option would be to write a quick AboutPreference that extends Preference and sets the title in the constructor. That way we wouldn't need to traverse the preferences here.